docs/sync.md

```
@@ -2,9 +2,7 @@
 
 All course content in PrairieLearn lives in a [Git](https://git-scm.com/) repository, typically hosted on [GitHub](https://github.com/). That repository is synced to/from PrairieLearn.
 
-<div style="max-width: 500px;" markdown>
-![](dev-guide/high-level.d2){pad="0"}
-</div>
+![](dev-guide/high-level.d2){pad="0" scale="1"}
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)